Doc/faq/general.rst

```diff
@@ -312,13 +312,13 @@ guaranteed that interfaces will remain the same throughout a series of bugfix
 releases.
 
 The latest stable releases can always be found on the `Python download page
-<https://www.python.org/downloads/>`_.  There are two production-ready version
-of Python: 2.x and 3.x, but the recommended one at this times is Python 3.x.
-Although Python 2.x is still widely used, `it will not be
-maintained after January 1, 2020 <https://www.python.org/dev/peps/pep-0373/>`_.
-Python 2.x was known for having more third-party libraries available, however,
-by the time of this writing, most of the widely used libraries support Python 3.x,
-and some are even dropping the Python 2.x support.
+<https://www.python.org/downloads/>`_.  There are two production-ready versions
+of Python: 2.x and 3.x, with the recommended one at this time being 3.x.
```

Comment:
Instead of 'at this time', you could consider 'as of October 2018', as that clarifies the time at which the text was written. Although it's also fair to assume that the recommendation will be valid until the rest of the text is also rewritten. (same for the 'at this time' at line 319)

Reply:
We could also just drop "at this time", since the recommended version is going to be 3.x in the foreseeable future.